pkg/repo/fs.go

```diff
@@ -82,39 +85,43 @@ func (fsr *FsRepo) Version() (int, error) {
 }
 
 // Init initializes a new repo, returning an error if the repo already exists.
-func (fsr *FsRepo) Init(c config.ReadWriter) error {
+func (fsr *FsRepo) Init(cfg types.Bacalhau) error {
 	if exists, err := fsr.Exists(); err != nil {
 		return err
 	} else if exists {
 		return fmt.Errorf("cannot init repo: repo already exists")
 	}
 
-	log.Info().Msgf("Initializing repo at '%s' for environment '%s'", fsr.path, config.GetConfigEnvironment())
```

Comment:
we no longer support a configuration environment, like Production, Staging, Development, and Testing. Instead there is a single default config.